src/main/java/org/folio/circulation/resources/handlers/error/CirculationErrorType.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.circulation.resources.handlers.error;
+
+public enum CirculationErrorType {
+  INVALID_ITEM_ID,
+  INVALID_PATRON_GROUP_ID,
+  INVALID_STATUS,
+  INVALID_PROXY_RELATIONSHIP,
+  INVALID_PICKUP_SERVICE_POINT,
+
+  FAILED_TO_FETCH_ITEM,
+  FAILED_TO_FETCH_USER,
+  FAILED_TO_FETCH_PROXY_USER,
+  FAILED_TO_FETCH_SERVICE_POINT,
+  FAILED_TO_FETCH_LOAN,
+  FAILED_TO_FETCH_REQUEST_QUEUE,
+  FAILED_TO_FETCH_REQUEST_POLICY,
+  FAILED_TO_FETCH_TIME_ZONE_CONFIG,
+  FAILED_TO_FETCH_PROXY_RELATIONSHIP,
+
+  ITEM_DOES_NOT_EXIST,
+  ITEM_ALREADY_REQUESTED_BY_SAME_USER,
+  ITEM_ALREADY_LOANED_TO_SAME_USER,
+
+  USER_DOES_NOT_EXIST,
+  USER_IS_INACTIVE,
+  USER_IS_BLOCKED_MANUALLY,
+  USER_IS_BLOCKED_AUTOMATICALLY,
+
+  REQUESTING_DISALLOWED_BY_POLICY,
+  REQUESTING_DISALLOWED,
+
+  USER_VALIDATION_FAILED,
+  ITEM_VALIDATION_FAILED,
+  AUTOMATED_BLOCKS_VALIDATION_FAILED,
+  MANUAL_BLOCKS_VALIDATION_FAILED
```

Comment:
My goal was to visually separate validation errors from non-validation errors, then group them further by failure reason or or validated entity. If that's confusing, I can pull them all together.

Reply:
I'm ok with logically splitting things up like this (when the types can't be separated).

I think a comment at the top of each group could explain the reasoning for the grouping that would allow other folks to understand and maintain those groupings.

Reply:
This enum now contains validation errors only, so I removed all grouping.